web/views.py

```
@@ -4681,55 +4690,78 @@ def donation_webhook(request):
 
     try:
         event = stripe.Webhook.construct_event(payload, sig_header, settings.STRIPE_WEBHOOK_SECRET)
-    except ValueError:
-        # Invalid payload
+        logger.info(f"Received Stripe webhook: {event.type}")
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)